pkg/jobs/trigger_interval.go

```diff
@@ -0,0 +1,69 @@
+package jobs
+
+import (
+	"errors"
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// IntervalTrigger implements the @interval trigger. It triggers a job on a
+// fixed period of time.
+type IntervalTrigger struct {
+	du   time.Duration
+	in   *TriggerInfos
+	done chan struct{}
+}
+
+// NewIntervalTrigger returns a new instace of IntervalTriven given the
+// specified options.
+func NewIntervalTrigger(infos *TriggerInfos) (*IntervalTrigger, error) {
```

Comment:
Yes... indeed. I should fix that or make it clear on the documentation that the `@interval` has no state.

However, I think the `@cron` scheduler should be more appropriate for these type of recurring tasks.